### PR TITLE
Chore: Plugin website build: Decrease commit size

### PR DESCRIPTION
**Pull request by personalizedrefrigerator**
# Summary

- Restores the deletion of `docs/plugins` in the Joplin Website CI before committing. As `docs/plugins` will be created just after the `Auto-updated` commit, this should significantly decrease the size of commits.

### CODE REVIEW

File: packages/tools/release-website.sh [hunk 1]
```diff
@@ -71,7 +71,13 @@ cd "$JOPLIN_ROOT_DIR"
 CROWDIN_PERSONAL_TOKEN="$CROWDIN_PERSONAL_TOKEN" yarn crowdinDownload
 yarn buildWebsite
 
+
 cd "$JOPLIN_WEBSITE_ROOT_DIR"
+
+# yarn buildWebsite removes docs/plugins. Restoring the original version
+# significantly decreases the size of commits.
+git restore docs/plugins || true
```
Comment (personalizedrefrigerator) [inline packages/tools/release-website.sh:79]:
```suggestion
git restore docs/plugins
```

Comment (laurent22) [reply packages/tools/release-website.sh:79]: What is the `|| true` for? I guess for `git commit` is that if there's no changes it returns an error (which we should ignore)?

Comment (personalizedrefrigerator) [reply packages/tools/release-website.sh:79]: That was the original logic (or if the changes were already staged by `buildWebsite`). However, there should always be changes as `buildWebsite` seems to remove the `plugins` directory.

**Comment by personalizedrefrigerator** [comment]
This won't correctly handle deletions.

**Pull request closed without merging**